apps/ewallet_db/lib/ewallet_db/auth_token.ex

```diff
@@ -190,6 +217,7 @@ defmodule EWalletDB.AuthToken do
   end
 
   # Expires the given token.
+  @spec expire(String.t(), atom(), any()) :: {:ok, %__MODULE__{}} | {:error, Changeset.t()}
```

Comment:
```suggestion
  @spec expire(String.t(), atom(), any()) :: {:ok, %__MODULE__{}} | {:error, Ecto.Changeset.t()}
```

Because `Ecto.Changeset` is not aliased